jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/websocket/WebSocketConstants.java

```diff
@@ -8,14 +8,54 @@
  * SPDX-License-Identifier: EPL-2.0 
  * 
  * Contributors: 
- *     Giancarlo Pernudi Segura - initial API and implementation 
+ *     Giancarlo Pernudi Segura - initial API and implementation
+ *     Lidia Ataupillco Ramos
  *******************************************************************************/
 
 package org.eclipse.lsp4jakarta.jdt.core.websocket;
 
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
 import org.eclipse.lsp4j.DiagnosticSeverity;
 
 public class WebSocketConstants {
     public static final String DIAGNOSTIC_SOURCE = "jakarta-websocket";
-    public static final DiagnosticSeverity SEVERITY = DiagnosticSeverity.Error;
+
+    public static final DiagnosticSeverity ERROR = DiagnosticSeverity.Error;
+    public static final DiagnosticSeverity WARNING = DiagnosticSeverity.Warning;
+
+    public static final String DIAGNOSTIC_ERR_MSG = "Cannot calculate WebSocket diagnostics";
+
+    public static final String DIAGNOSTIC_PATH_PARAMS_ANNOT_MISSING = "Parameter is missing annotation @PathParams.";
```

Comment:
Can we rephrase this diagnostic message to be more specific? Indicating that parameters of type String, any Java primitive types or boxed version thereof must be annotated with `@PathParams`

Reply:
will do that!